app/components/download-graph.js

```diff
@@ -106,7 +106,14 @@ export default class DownloadGraph extends Component {
       }
     });
 
-    let versionsList = semverSort([...versions]);
+    let versionsList = [...versions];
+    try {
+      semverSort(versionsList);
```

Comment:
does this sort the array in-place?

Reply:
I think it does, unless I'm misunderstanding something
https://github.com/rust-lang/crates.io/blob/master/app/components/download-graph.js#L8
https://github.com/npm/node-semver/blob/master/functions/sort.js#L2
https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Array/sort
> The sort() method sorts the elements of an array in place

Reply:
I also checked how the code behaves when I comment out semverSort and it seems to confirm that

Reply:
okay, great, thanks :)